cli/command/configure/cmd.go

```diff
@@ -0,0 +1,142 @@
+package configure
+
+import (
+	"errors"
+	"fmt"
+	"github.com/rs/zerolog/log"
+	"github.com/saucelabs/saucectl/cli/command"
+	"github.com/saucelabs/saucectl/cli/credentials"
+	"github.com/spf13/cobra"
+	"github.com/tj/survey"
+	"os"
+	"regexp"
+)
+
+var (
+	configureUse     = "configure"
+	configureShort   = "Configure your Sauce Labs credentials"
+	configureLong    = `Persist locally your Sauce Labs credentials`
+	configureExample = "saucectl configure"
+	cliUsername      = ""
+	cliAccessKey     = ""
+)
+
+// Command creates the `new` command
```

Comment:
@FriggaHel should this be `// Command creates the `configure` command` ?

Reply:
Yep ! I'll correct that